http-clients/src/main/java/com/palantir/remoting/http/FeignClients.java

```diff
@@ -53,6 +53,21 @@ public static FeignClientFactory standard() {
     }
 
     /**
+     * Provides a {@link FeignClientFactory} compatible with jackson 2.4.
```

Comment:
The method still exists in Jackson 2.6, so not strictly true they need to use Jackson 2.4.